.github/workflows/per-pr.yml

```diff
@@ -26,19 +26,29 @@ jobs:
           version: v1.61
           args: --timeout 3m
 
-  # This checks that the flags in .env.holesky.example are valid and allow the proxy to start.
-  flags:
+  # This checks that the flags in .env.exampleV1.holesky are valid and allow the proxy to start.
+  flagsV1:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
           go-version: ${{ env.GO_VERSION}}
-      - name: Run flag test
-        run: ${{ github.workspace }}/scripts/test-proxy-startup-with-env-vars.sh
+      - name: Run flag test V1
+        run: ${{ github.workspace }}/scripts/test-proxy-startup-with-env-vars.sh .env.exampleV1.holesky
 
-  # This ensures that std output generated when running binary with `--help` is reflected in
-  # in docs/help_out.txt
+  # This checks that the flags in .env.exampleV2.holesky are valid and allow the proxy to start.
+  flagsV2:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION}}
```

Comment:
use a matrix strategy instead: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#using-a-matrix-strategy

Reply:
[eb93cad](https://github.com/Layr-Labs/eigenda-proxy/pull/349/commits/eb93cad30057e88ea3b6cc5d47fa0c89c52ccb9e)